File: lang_java/ProcessingUnit.java
class ProcessingUnit extends Komponen{

    private int jumlahCore;
    private double kecepatanGHz;
    private String tipeUnit;

    public ProcessingUnit(){
    }

    public ProcessingUnit(String merk, String nama, int jumlahCore, double kecepatanGHz, String tipeUnit){
        super(merk, nama);
        this.jumlahCore = jumlahCore;
        this.kecepatanGHz = kecepatanGHz;
        this.tipeUnit = tipeUnit;
    }

    public void setJumlahCore(int jumlahCore){
        this.jumlahCore = jumlahCore;
    }

    public int getJumlahCore(){
        return jumlahCore;
    }

    public void setKecepatanGHz(double kecepatanGHz){
        this.kecepatanGHz = kecepatanGHz;
    }

    public double getKecepatanGHz(){
        return kecepatanGHz;
    }

    public void setTipeUnit(String tipeUnit){
        this.tipeUnit = tipeUnit;
    }

    public String getTipeUnit(){
        return tipeUnit;
    }

}

File: lang_py/ProcessingUnit.py
from Komponen import Komponen

class ProcessingUnit(Komponen):

    def __init__(self, merk, nama, jumlahCore, kecepatanGHz, tipeUnit):
        super().__init__(merk, nama)
        self.jumlahCore = jumlahCore
        self.kecepatanGHz = kecepatanGHz
        self.tipeUnit = tipeUnit

    def setJumlahCore(self, jumlahCore):
        self.jumlahCore = jumlahCore

    def getJumlahCore(self):
        return self.jumlahCore
    
    def setKecepatanGHz(self, kecepatanGHz):
        self.kecepatanGHz = kecepatanGHz

    def getKecepatanGHz(self):
        return self.kecepatanGHz
    
    def setTipeUnit(self, tipeUnit):
        self.tipeUnit = tipeUnit

    def getTipeUnit(self):
        return self.tipeUnit